### ISSUE TITLE
I just upload a cat image, but the result is 100% dog.

### ISSUE THREAD

**Issue by watershade** [leemengtaiwan/cat-recognition-app#4]
I just upload a cat image, but the result is 100% dog.
Photo 48
But I don't know how to feedback. Sorry about it.

**Comment by leemengtaiwan** [comment]
@watershade 
Thanks for your feedback!
Although it might be insufficient data augmentation problem (in plain English, not enough training data), if you can paste the picture URL you used to test, maybe we can take a deeper look at it :)
